test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -71,3 +72,30 @@
         (is (nil? (get catalog "code_id")))
         (is (logged? #"Non-zero exit code returned while running" :error))
         (is (logged? #"Executed an external process which logged to STDERR: production" :warn)))))))
+
+(deftest ^:integration static-file-content-endpoint-test
+  (logging/with-test-logging
+    (testing "the /static_file_content endpoint successfully streams file content"
+      (bootstrap/with-puppetserver-running
+        app
+        {:jruby-puppet
+         {:max-active-instances num-jrubies}
+         :versioned-code
+         {:code-content-command (script-path "echo")}}
+        (let [response (http-client/get "https://localhost:8140/puppet/v3/static_file_content/foo/bar/?code-id=foobar&environment=test"
```

Comment:
@fpringvaldsen :+1: on `code_id`

Also the request path looks correct, though to be a bit more specific, I'm assuming the `path` will be rooted in a specific per-environment directory and will be taken directly from the `content_uri` parameter described in https://docs.google.com/document/d/1bZU1WJAvVQfyoBHf-5kXGYivg7r_ul5o9boimCdpYuM/edit#heading=h.lhexacaboszr. So I'm thinking it'd be something like:

https://localhost:8140/puppet/v3/static_file_content/modules/ntp/files/ntpd.conf`

I also believe the agent will send the `environment` query parameter (duplicating information in `code_id`), because it's hard to get the agent to not send it.